data/outport/outportBlock.proto

```diff
@@ -26,6 +26,8 @@ message OutportBlock {
   repeated uint64              SignersIndexes         = 8 [(gogoproto.jsontag) = "signersIndexes,omitempty"];
   uint64                       HighestFinalBlockNonce = 9 [(gogoproto.jsontag) = "highestFinalBlockNonce"];
   bytes                        HighestFinalBlockHash  = 10 [(gogoproto.jsontag) = "highestFinalBlockHash,omitempty"];
+  uint64                       LeaderIndex            = 11 [(gogoproto.jsontag) = "leaderIndex"];
```

Comment:
Shouldn't this be omitEmpty as well? for before activation

Reply:
Before activation, I think we can let it without omitempty to see the leader on index 0